packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/UpgradeRolesForm/UpgradeRolesForm.tsx

```diff
@@ -19,14 +19,23 @@ type UpgradeRolesFormProps = {
   setSearchTerm: (v: string) => void;
 };
 
+type UserDisplayRow = {
+  profile_name: string | null | undefined;
+  user_id: number;
+  avatar_url: string | null | undefined;
+  role: string;
+  address_id: number;
+  address: string;
```

Comment:
we usually use camel case for component-related types

Reply:
The problem is that this data comes from the server, and normally the data returned from the server is snake case because it is being pulled from sql, which doesn't play nicely with camelCase (You need quotation marks around it). So the other alternative here is if we had some layer in between that turned these from snake to camel case, but maybe that would make the code a little messy, since we need to deal with the snake cased variables anyways.

Curious what you think

Reply:
makes sense, we already have a lot of messy `scripts/models` code. I think its fine for now. Thanks!